sdk/scheduler/src/test/resources/invalid-pod-name.yml

```diff
@@ -6,7 +6,7 @@ pods:
     resource-sets:
       meta-data-resources:
         cpus: 0.1
-        mem: 512
+        memory: 512
```

Comment:
Are the changes you made to the tests sufficient to make them fail now if we put in unknown properties?